randovania/games/am2r/json_data/Distribution Center.txt

```diff
@@ -1598,48 +1570,30 @@ Extra - map_name: rm_a5b15
 > Dock to Facility Storage Tower East; Heals? False
   * Layers: default
   * Open Passage to Facility Storage Tower East/Dock to Spiky Space Jump Trial Entrance
-  > Dock to Spiky Space Jump Trial
-      All of the following:
-          Gravity Suit and Speed Booster and Shinesparking Tricks (Beginner)
-          Any of the following:
-              # Climb Requirements
-              Hi-Jump Boots or Power Grip or Space Jump or Walljump (Beginner)
-  > Bottom
-      Any of the following:
-          Power Grip or Movement (Beginner) or Walljump (Beginner)
-          Gravity Suit and Power Grip Wall
-          All of the following:
-              # IBJ
-              Infinite Bomb Jumping (Intermediate) and Can Use Bombs
-          All of the following:
-              # CBJ
-              Charged Bomb Jump (Beginner) and Can Use Charged Bomb Jump
+  > Center
+      Movement (Beginner) or Underwater Power Grip Wall
 
 > Dock to Spiky Space Jump Trial; Heals? False
```

Comment:
This dock should be able to reach the tank with just speedboosting. In the weird case where you have entrance rando, and this would be your power grip for example, or something.